wallet/pkgs/model_viewer_plus.dart-master/lib/src/model_viewer_plus_stub.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+
+import 'model_viewer_plus.dart';
+
+class ModelViewerState extends State<ModelViewer> {
```

Comment:
We dont need this file.

Reply:
We need it, as it is using in the package